src/testing/vortex/supervisor.zig

```diff
@@ -678,18 +676,18 @@ pub fn comma_separate_ports(comptime ports: []const u16) []const u8 {
     return out;
 }
 
-// Create a new Vortex-specific temporary directory. Caller owns the returned memory.
-fn create_tmp_dir(allocator: std.mem.Allocator) ![]const u8 {
+// Create a new Vortex-specific temporary directory.
+fn create_tmp_dir(absolute_path_buffer: []u8) ![]const u8 {
```

Comment:
<3, good to get rid of an allocator here!

Reply:
Couldn't resist!